src/utils/logging.rs

```diff
@@ -7,44 +7,42 @@ use std::{
 
 use crate::utils::consts::{MOSAIC_TMP_LOG_DIR, MOSAIC_TMP_LOG_FILE};
 
-fn atomic_create_file(file_name: &str) {
-    let _ = fs::OpenOptions::new().create(true).open(file_name);
+fn atomic_create_file(file_name: &str) -> io::Result<fs::File> {
+    fs::OpenOptions::new().create(true).open(file_name)
```

Comment:
@5c077m4n - the previous way was actually intentional. This is "atomic" create file, meaning that it's okay for it to fail if the file already exists.

Reply:
Reverted